### PR TITLE
Move appliance params parsing into provider class

### PR DESCRIPTION
**Pull request by mkasztelnik**

### CODE REVIEW

**Comment by mkasztelnik** [comment]
@dice-cyfronet/atmo-dev-team please take a look.

**Comment by mkasztelnik** [comment]
@dice-cyfronet/atmo-dev-team any comments/remarks?

File: app/services/atmosphere/appliance_creator.rb [hunk 1]
```diff
@@ -1,114 +1,25 @@
...
-
-      params.permit(allowed_params)
-    end
-
-    def allowed_compute_sites
-      if params[:compute_site_ids].blank?
-        ComputeSite.active
-      else
-        ComputeSite.where(id: params[:compute_site_ids], active: true)
+      pprovider = ApplianceParams.
```
Comment (nowakowski) [inline app/services/atmosphere/appliance_creator.rb]: Why ppppprovider? :)

Comment (mkasztelnik) [inline app/services/atmosphere/appliance_creator.rb]: Shortcut from parameter provider :smile:

**Comment by nowakowski** [comment]
:+1:

**Pull request merged**